distutils deprecated in python 3.10

I know this is early but on `python 3.10`, I notice the following:

```
/opt/hostedtoolcache/Python/3.10.2/x64/lib/python3.10/site-packages/scooby/__init__.py:18: in <module>
    from scooby.knowledge import (  # noqa
/opt/hostedtoolcache/Python/3.10.2/x64/lib/python3.10/site-packages/scooby/knowledge.py:14: in <module>
    import distutils.sysconfig as sysconfig
/opt/hostedtoolcache/Python/3.10.2/x64/lib/python3.10/distutils/__init__.py:19: in <module>
    warnings.warn(_DEPRECATION_MESSAGE,
E   DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
```

Related to https://github.com/pyvista/pyvistaqt/pull/147

--- a/scooby/knowledge.py
+++ b/scooby/knowledge.py
@@ -11,7 +11,7 @@ It also checks and stores mandatory additional information, if possible, such
 as available RAM or MKL info.
 
 """
-import distutils.sysconfig as sysconfig
+import sysconfig
 import os
 from pathlib import Path
 import platform
@@ -111,8 +111,10 @@ def in_ipykernel():
 
 def get_standard_lib_modules():
     """Returns a set of the names of all modules in the standard library"""
+    site_path = sysconfig.get_path('stdlib')
+
     if getattr(sys, 'frozen', False):  # within pyinstaller
-        lib_path = os.path.join(sysconfig.get_python_lib(standard_lib=True), '..')
+        lib_path = os.path.join(site_path, '..')
         if os.path.isdir(lib_path):
             names = os.listdir(lib_path)
         else:
@@ -125,7 +127,7 @@ def get_standard_lib_modules():
         stdlib_pkgs = set(stdlib_pkgs)
 
     else:
-        names = os.listdir(sysconfig.get_python_lib(standard_lib=True))
+        names = os.listdir(site_path)
 
         stdlib_pkgs = set([name if not name.endswith(".py") else name[:-3] for name in names])
 

--- a/scooby/knowledge.py
+++ b/scooby/knowledge.py
@@ -11,11 +11,11 @@ It also checks and stores mandatory additional information, if possible, such
 as available RAM or MKL info.
 
 """
-import sysconfig
 import os
 from pathlib import Path
 import platform
 import sys
+import sysconfig
 
 try:
     import psutil

